### ISSUE TITLE
NS metadata is lost or breaks rewrite completely

### ISSUE THREAD

**Issue by axelarge** [oakmac/standard-clojure-style-js#113]
![CleanShot 2024-10-09 at 13 56 40@2x](https://github.com/user-attachments/assets/63f13efb-e211-4dfb-92ca-99b37f64687e)
![CleanShot 2024-10-09 at 13 56 17@2x](https://github.com/user-attachments/assets/ebc757b0-bf23-4daf-9321-d7fd0105e928)
![CleanShot 2024-10-09 at 13 55 32@2x](https://github.com/user-attachments/assets/0d807405-3dca-452c-a29b-6af6a96e922d)

Just to show that these are all valid:
```clojure
user=> (ns ^:kw-flag my.ns1)
nil
my.ns1=> (meta *ns*)
{:kw-flag true}

my.ns1=> (ns ^{:meta-map 123} my.ns2)
nil
my.ns2=> (meta *ns*)   
{:meta-map 123}

my.ns2=> (ns my.ns3 {:trailing :map})
nil
my.ns3=> (meta *ns*)
{:trailing :map}
my.ns3=> 
```

**Comment by oakmac** [comment]
Related to [Issue #55](https://github.com/oakmac/standard-clojure-style-js/issues/55)

**Comment by oakmac** [comment]
Thank you for the report!

Fixed with [PR-101](https://github.com/oakmac/standard-clojure-style-js/pull/101)

**Comment by oakmac** [comment]
This fix is available with v0.7.0
